tools/ci/azure/tox_pytest.yml

```diff
@@ -0,0 +1,18 @@
+parameters:
+  directory: ''
+  toxenv: 'ALL'
+
+steps:
+- template: pip_install.yml
+  parameters:
+    packages: tox virtualenv
```

Comment:
Nit: `tox` is enough. It depends on `virtualenv`.

Reply:
Nice, thanks!